resources/jenkins/release.sh

```diff
@@ -3,43 +3,37 @@ set -e
 
 # This script should be run after the build/tests have successfully completed
 
-MAJOR_MINOR=$1
-
 # Prune any local tags that aren't on the remote
 git fetch --prune origin "+refs/tags/*:refs/tags/*"
 
-DESCRIBE=`git describe --tags --always`
-REV=$(git log --pretty=format:'%h' -n 1)
+cd $WORKSPACE
 
-CURR_VERSION=`echo $DESCRIBE | awk '{split($0,a,"-"); print a[1]}'`
-echo "Got current git version $CURR_VERSION"
-MAJOR_VERSION=`echo $CURR_VERSION | awk '{split($0,a,"."); print a[1]}'`
-MINOR_VERSION=`echo $CURR_VERSION | awk '{split($0,a,"."); print a[2]}'`
+# Let's get version from maven
+MVNVER=`xmllint --xpath "/*[local-name()='project']/*[local-name()='version']/text()" pom.xml`
+TAG_NAME="v${MVNVER/-SNAPSHOT/}"
+echo $TAG_NAME
```

Comment:
can we add some context to this print? like `echo "Current tag name: $TAG_NAME"`?